src/graphql/data/nft/Asset.ts

```diff
@@ -117,22 +116,46 @@ export function useAssetsQuery(
   last?: number,
   before?: string
 ) {
-  const queryData = useLazyLoadQuery<AssetQuery>(assetQuery, {
-    address,
-    orderBy,
-    asc,
-    filter,
-    first,
-    after,
-    last,
-    before,
+  const queryResult = useQuery(assetQuery, {
+    variables: {
+      address,
+      orderBy,
+      asc,
+      filter,
+      first,
+      after,
+      last,
+      before,
+    },
   })
-  const { data, hasNext, loadNext, isLoadingNext } = usePaginationFragment<AssetPaginationQuery, any>(
-    assetPaginationQuery,
-    queryData
-  )
+  const { loading, data, fetchMore, startPolling } = queryResult
+
+  startPolling(999)
+
+  const loadMore = () => {
+    fetchMore({
+      variables: {
+        after: data?.nftAssets?.pageInfo?.endCursor,
+      },
+    })
+  }
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
```

Comment:
doesn't the interface codebase have a `useInterval` hook?

Reply:
https://github.com/Uniswap/interface/blob/caa0a2967d169dbe75a42a9a2ef3ba8b736614ba/src/lib/hooks/useInterval.ts#L1